solutions/devsprint-bruno-ramos-5/GitHubApp/Screens/List/ListView.swift

```diff
@@ -79,18 +80,25 @@ extension ListView {
     }
 }
 
-extension ListView: UITableViewDataSource {
+extension ListView: UITableViewDataSource, UITableViewDelegate {
 
     public func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
 
         return self.listItems.count
     }
+    
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return 62
+    }
 
     public func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
 
-        let cell = tableView.dequeueReusableCell(withIdentifier: self.listViewCellIdentifier)!
-        cell.textLabel?.text = self.listItems[indexPath.row]
-        return cell
+        let cell = tableView.dequeueReusableCell(withIdentifier: listViewCellIdentifier, for: indexPath) as? RepositoryViewCell
```

Comment:
Para não precisar trabalhar com a cell opcional em todo o contexto do método, podemos fazer o unwrap da cell no ínicio do método e no caso que não for atendido, podemos retornar uma cell vazia, porém em um projeto real, poderíamos fazer o disparo de um log de erro.
```suggestion
        guard let cell = tableView.dequeueReusableCell(withIdentifier: listViewCellIdentifier, for: indexPath) as? RepositoryViewCell else { 
        // Log para algum provedor (Ex: Logz.io)
        return UITableViewCell() 
}
```